project.go

```diff
@@ -0,0 +1,391 @@
+// Copyright 2016 EF CTX. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/tsuru/gnuflag"
+	"github.com/tsuru/tsuru/cmd"
+	tsuruerrors "github.com/tsuru/tsuru/errors"
+)
+
+type projectCreate struct {
+	fs       *gnuflag.FlagSet
+	name     string
+	platform string
+	team     string
+	plan     string
```

Comment:
@fsouza, @aflag, @Lepthy: Do you think it makes sense, within the same project, to have different `plans` per environment?
Eg. a certain API in dev & QA may require only 128MB memory, but we may want to be safe in prod and have 512MB.